packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,153 @@
+default.target=.
+
+## all templates to apply on target deployment env
+nuxeo.template.includes=default,s3binaries,postgresql,explorer-sitemode,explorer-virtualadmin
+# TODO: check logs config with 'docker' or 'docker-json' template
+
+## explorer confs mapping
+
+org.nuxeo.apidoc.apidocAdmin.password=${env:ORG_NUXEO_APIDOC_APIDOCADMIN_PASSWORD}
+
+
+## default confs mapping
+
+nuxeo.data.dir=${env:NUXEO_DATA_DIR:nxserver/data}
+nuxeo.server.dir=${env:NUXEO_SERVER_DIR:nxserver}
```

Comment:
@nuxeo/platform please tell if these two are handled in the nuxeo docker image

Reply:
I would expect the nuxeo.data.dir to be handled by the default docker image.
For the nuxeo.server.dir, there should be no need to set it specifically for the explorer.

Reply:
data.dir and co confirmed to be entrypoints on the base nuxeo image --> removed